src/features/listing-builder/components/Form/EligibilityQuestions/Sheet.tsx

```
@@ -0,0 +1,157 @@
+  const [open, setOpen] = useState(false);
+
+  const type = useWatch({
+    control: form.control,
+    name: 'type',
+  });
+
+  const hasEligibilityQsErrors = useMemo(() => {
+    const errors = form.formState.errors;
+    return (
+      errors.eligibility &&
+      errors?.eligibility?.some?.((question) => !!question)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)